src/postalcode.js

```diff
@@ -0,0 +1,71 @@
+define([
+	"cldr",
+	"./core",
+	"./common/validate/cldr",
+	"./common/validate/default-locale",
+	"./common/validate/parameter-presence",
+	"./common/validate/parameter-type/string",
+
+	"cldr/event",
+	"cldr/supplemental"
+], function( Cldr, Globalize, validateCldr, validateDefaultLocale,
+	validateParameterPresence, validateParameterTypeString ) {
+
+/**
+ * .validatePostalCode( value )
+ *
+ * @value [String]
+ *
+ * TRUE if the value is matched with postalcode pattern given the locale.
+ * FALSE if the value is not matched with postalcode pattern given the locale.
+ *
+ * Return the boolean.
+ */
+Globalize.validatePostalCode =
+Globalize.prototype.validatePostalCode = function( value ) {
+	validateParameterPresence( value, "value" );
+    validateParameterTypeString( value, "value" );
+	return this.postalCodeValidator()( value );
+};
+
+/**
+ * .postalCodeValidator()
+ *
+ * Return a validatePostalCode function (of the form below).
+ *
+ * fn( value )
+ *
+ * @value [String]
+ *
+ *
+ * TRUE if the value is matched with postalcode pattern given the locale.
+ * FALSE if the value is not matched with postalcode pattern given the locale.
```

Comment:
This makes it look like `FALSE` is some kind of special constant. Let's stick with `true` and `false`. Could also simplify this into a single sentence: "Returns true if the value matches the postalcode pattern for the given locale, otherwise returns false."